src/backend/effects/builtin/chat.js

```diff
@@ -16,26 +16,42 @@ const effect = {
     <eos-chatter-select effect="effect" title="Chat as"></eos-chatter-select>
 
     <eos-container header="Message To Send" pad-top="true">
-        <textarea ng-model="effect.message" class="form-control" name="text" placeholder="Enter message" rows="4" cols="40" replace-variables></textarea>
+        <firebot-input 
+            model="effect.message" 
+            use-text-area="true"
+            placeholder-text="Enter message"
+            rows="4"
+            cols="40"
+        />
         <div style="color: #fb7373;" ng-if="effect.message && effect.message.length > 500">Chat messages cannot be longer than 500 characters. This message will get automatically chunked into multiple messages if it's too long after all replace variables have been populated.</div>
         <div style="display: flex; flex-direction: row; width: 100%; height: 36px; margin: 10px 0 10px; align-items: center;">
-            <label class="control-fb control--checkbox" style="margin: 0px 15px 0px 0px"> Whisper
-                <input type="checkbox" ng-init="whisper = (effect.whisper != null && effect.whisper !== '')" ng-model="whisper" ng-click="effect.whisper = ''">
-                <div class="control__indicator"></div>
-            </label>
-            <div ng-show="whisper">
-                <div class="input-group">
-                    <span class="input-group-addon" id="chat-whisper-effect-type">To</span>
-                    <input ng-model="effect.whisper" type="text" class="form-control" id="chat-whisper-setting" aria-describedby="chat-text-effect-type" placeholder="Username" replace-variables>
-                </div>
+            <firebot-checkbox 
+                label="Use '/me'" 
+                tooltip="Applies Italics to your Chat Message or your Chat Color if used in a Whisper" 
+                model="effect.me"
+                style="margin: 0px 15px 0px 0px"
+            />
+            <firebot-checkbox 
+                label="Whisper"
+                model="effect.whisper"
+                style="margin: 0px 15px 0px 0px"
+            />
+            <div ng-show="effect.whisper">
+                <firebot-input 
+                    input-title="To"
+                    model="effect.whisperTarget" 
+                    placeholder-text="Username"
+                />
             </div>
```

Comment:
Thanks for refactoring old html into the newer components! This change here however would constitute a breaking change for all chat effects created before this that had a whisper username set. This is because "whisperTarget" is a new property that didnt exist before, meaning no existing chat effect will have it set. 

Here's some tweaks we can make to prevent it from being a breaking change. 

1. For the `optionsController` in this file, we can emulate the same logic that was within the old "ng-init" property
```js
optionsController: ($scope) => {
    $scope.showWhisperInput = $scope.effect.whisper != null && $scope.effect.whisper !== ''
},
```

2. Then update the Whipser checkbox to point the model property to this new scope value we created in the controller:
```html
<firebot-checkbox 
    label="Whisper"
    model="showWhisperInput"
    style="margin: 0px 15px 0px 0px"
    ng-click="effect.whisper = ''"
/>
```

3. Update the firebot-input for the whisper username to point back to the original property:
```html
<firebot-input 
    input-title="To"
    model="effect.whisper"
    placeholder-text="Username"
/>
```

Reply:
Thank you for this! I've gone ahead and made these changes and pushed them up 🙏